### ISSUE TITLE
feature: Add an action to clean all inserted logs by the plugin

### ISSUE THREAD

**Issue by karamanliev** [Goose97/timber.nvim#3]
### Did you check the docs?

- [X] I have read all the docs

### Is your feature request related to a problem? Please describe.

It's just nice to have the option to remove all the inserted logs when you're done with debugging.

### Describe the solution you'd like

Add `require("timber.actions").clear_logs({global = false})`. If global `true` delete everywhere, if `false` delete only in the current file.

### Describe alternatives you've considered

I can add an command by myself, which finds the logs and clears them by a regex or something, but I really believe that a plugin which inserts logs, should be able to clear/remove them.

### Additional context

Thanks for the plugin, it's really nice!

**Comment by Goose97** [comment]
@karamanliev This feature is implemented in #5. Thank you for the suggestion, I adopted your interface with a slight adjustment:

```lua
require("timber.actions").clear_log_statements({ global = false })
```

However, beware of [some limitations](https://github.com/Goose97/timber.nvim/blob/a2faec8a7525d49a2e033ce54246cd50a4fb9021/doc/timber.nvim.txt#L245-L250):
- Some code formatters can automatically break up the log statements into multiple lines. In this case, theres' current no mechanism to keep track of the lines.
- We only clear from all loaded buffers, not all files. It means if you just open Neovim, and run the clear, log statements in existing files won't be cleared. To do this, we might need some project wide search and perform delete over it. That's too complicated for now

**Comment by karamanliev** [comment]
Nice, thank you! Most people need to clear logs from the current buffer only anyways :)

Thanks again for the plugin!
